src/app/admin/add-project/add-project.component.ts

```diff
@@ -7,9 +10,40 @@ import { Component, OnInit } from '@angular/core';
 })
 export class AddProjectComponent implements OnInit {
 
-  constructor() { }
+  constructor(
+    private fb: FormBuilder,
+    private http: HttpClient,
+    private router: Router
+  ) { }
 
   ngOnInit(): void {
   }
 
+  projForm = this.fb.group({
+    title: [''],
+    type: [''],
+    description:[''],
+  });
+
+  async onSubmit() {
+    let formData = {
+      title: this.projForm.get('title').value,
+      type: this.projForm.get('type').value,
+      description: this.projForm.get('description').value,
+      observations: [],
+    };
+
+    try {
+      const res = await this.http
+        .post('http://localhost:3000/project/create', formData)
```

Comment:
We probably want to abstract this into the projects service that I just created! I can do that once your pull request is merged in :)